docker/conf/docker-sync.yml.dist

```diff
@@ -20,3 +20,13 @@ syncs:
             'web',
             'vendor'
         ]
+
+    shopsys-framework-web-sync:
```

Comment:
docker-sync for windows is missing and sync_excludes contains duplicitly `docker` and `docker/` please see #35 and missing CHANGELOG.md